README.md

```diff
@@ -1,79 +1,66 @@
-
-[![Build Status](https://travis-ci.org/OfficeDev/microsoft-teams-library-js.svg?branch=master)](https://travis-ci.org/OfficeDev/microsoft-teams-library-js)
-[![Coverage Status](https://coveralls.io/repos/github/OfficeDev/microsoft-teams-library-js/badge.svg?branch=master)](https://coveralls.io/github/OfficeDev/microsoft-teams-library-js?branch=master)
+Welcome to the Teams client SDK monorepo! For breaking changes, please refer to our changelog in the monorepo root of the `2.0-preview` branch. This repository contains the core teams-client-sdk as well as tools and applications for analyzing and testing.
 
 ## Getting Started
 
-1. Clone the repo
-2. Run `yarn install` from repo root
-3. Run `yarn build` from repo root
-4. To run Unit tests, run `yarn test`
+ramourt 10 minutes ago •
+edited
+Member
 
-  TIP: whenever building or testing the Teams client SDK, you can run `yarn build` or `yarn test` from the packages/teams-js directory.
+We should keep this with the 4 instructions and the tip. Above the list of apps and under the coverage thing
 
-## Installation
+Jessica-Koch 13 seconds ago Author Member
 
-To install the latest 2.0 preview version:
+oh i totally agree, this was a mistake on my part
 
-### npm
+    Reply…
 
-`npm install --save @microsoft/teams-js@next`
+Resolve conversation
 
-### yarn
+1. Clone the repo
+2. Run `yarn install` from repo root
+3. Run `yarn build` from repo root
+4. To run Unit tests, run `yarn test`
 
-`yarn add @microsoft/teams-js@next`
+TIP: whenever building or testing the Teams client SDK, you can run `yarn build` or `yarn test` from the `packages/teams-js` directory.
 
-### Production
+This JavaScript library is part of the [Microsoft Teams developer platform](https://developer.microsoft.com/microsoft-teams/). See full [SDK reference documentation](https://docs.microsoft.com/en-us/javascript/api/overview/msteams-client).
 
-You can access [these files on unpkg](https://res.cdn.office.net/teams-js/2.0.0-beta.2/js/MicrosoftTeams.min.js), download them, or point your package manager to them.
+[![Build Status](https://travis-ci.org/OfficeDev/microsoft-teams-library-js.svg?branch=2.0-preview)](https://travis-ci.org/OfficeDev/microsoft-teams-library-js)
+[![Coverage Status](https://coveralls.io/repos/github/OfficeDev/microsoft-teams-library-js/badge.svg?branch=2.0-preview)](https://coveralls.io/github/OfficeDev/microsoft-teams-library-js?branch=2.0-preview)
 
-## Usage
+# Packages
 
-### As a package
+Contain the core exports for the repository.
 
-Install either using npm or yarn.
+### [Teams-Js](./packages/teams-js)
 
-**If you are using any dependency loader** such as [RequireJS](http://requirejs.org/) or [SystemJS](https://github.com/systemjs/systemjs) or module bundler such as [browserify](http://browserify.org/), [webpack](https://webpack.github.io/), you can use `import` syntax to import specific modules. For e.g.
+Used to integrate custom services and applications with Teams
```

Comment:
Since v2 supports extending apps to Office and Outlook, I'm considering whether we should add them here, e.g.,

"Used to integrate custom services and applications with Teams, Outlook, and Office".

Reply:
I think it'd be good to add, thanks! And small-nit-ahead-of-time, can you please end the sentence with a period to be consistent with the other places? :)

Reply:
i think that's a good thing to note and add